### PR TITLE
Ext hcp vault

### PR DESCRIPTION
**Pull request by IrAriasKC**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [✔️] Did you check you NRQL syntax? - Does it work?
- [✔️] Did you include an InstallPlan and Documentation reference?
- [✔️] Are all documentation links publically accessible?
- [✔️] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [✔️] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [✔️] Does the PR contain a screenshot for each of your dashboards?
- [✔️] Do your screenshots show data?
- [✔️] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [✔️] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by tabathadelane** [comment]
Hello, @IrAriasKC  ! Please sign the CLA linked above so we can run our validation workflows on your PR.

File: install/third-party/ext-hcp_vault/install.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+id: ext-hcp_vault
```
Comment (aswanson-nr) [inline install/third-party/ext-hcp_vault/install.yml:1]:
```suggestion
id: ext-hcp-vault
```

File: quickstarts/ext-hcp_vault/config.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: ext-hcp_vault
```
Comment (aswanson-nr) [inline quickstarts/ext-hcp_vault/config.yml:2]:
```suggestion
slug: ext-hcp-vault
```

File: quickstarts/ext-hcp_vault/config.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+# Displayed in the UI (required)
+title: ext-hcp_vault
```
Comment (aswanson-nr) [inline quickstarts/ext-hcp_vault/config.yml:5]: This is the display/title that is display within the UI, would you like to update this to a title case display name?

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @IrAriasKC, this quickstart looks fantastic! I noticed a couple of issues with the file paths, we enforce that they are lowercase and hyphenated. Could you update the folder names for the alerts, dashboard, install plan, and quickstart to using hypens instead of underscores? 
I commented one a few other places that need to be updated as well!

File: quickstarts/ext-hcp_vault/config.yml (hunk 3 above)
Comment (IrAriasKC) [reply quickstarts/ext-hcp_vault/config.yml:5]: @aswanson-nr , I've performed the suggested changes, thanks

File: quickstarts/ext-hcp_vault/config.yml (hunk 2 above)
Comment (IrAriasKC) [reply quickstarts/ext-hcp_vault/config.yml:2]: I've performed the suggested changes, thanks

File: install/third-party/ext-hcp_vault/install.yml (hunk 1 above)
Comment (IrAriasKC) [reply install/third-party/ext-hcp_vault/install.yml:1]: I've performed the suggested changes, thanks

File: quickstarts/ext-hcp-vault/config.yml [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+documentation:
+  - name: HashiCorp Vault installation docs
+    description: |
+      The HCP Vault quickstart allows you to get visibility into the performance of your instances.
+    url: >-
+      https://www.vaultproject.io/docs/configuration/telemetry
+
+installPlans:
+  - third-party-ext-hcp_vault
```
Comment (aswanson-nr) [inline quickstarts/ext-hcp-vault/config.yml:76]:
```suggestion
  - ext-hcp-vault
```

File: quickstarts/ext-hcp-vault/config.yml [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+alertPolicies:
+  - ext-hcp_vault
```
Comment (aswanson-nr) [inline quickstarts/ext-hcp-vault/config.yml:79]:
```suggestion
  - ext-hcp-vault
```

File: quickstarts/ext-hcp-vault/config.yml [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+dashboards:
+  - ext-hcp_vault
```
Comment (aswanson-nr) [inline quickstarts/ext-hcp-vault/config.yml:81]:
```suggestion
  - ext-hcp-vault
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Just need to update the links in the quickstart config, I have attached comments! Then this should be good to merge!

File: quickstarts/ext-hcp-vault/config.yml (hunk 6 above)
Comment (IrAriasKC) [reply quickstarts/ext-hcp-vault/config.yml]: Hi @aswanson-nr , we have performed the recommended changes

File: alert-policies/ext-hcp-vault/handle-login-request.yml [hunk 7]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (aswanson-nr) [inline alert-policies/ext-hcp-vault/handle-login-request.yml:1]:
```suggestion
```

File: alert-policies/ext-hcp-vault/handle-request.yml [hunk 8]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (aswanson-nr) [inline alert-policies/ext-hcp-vault/handle-request.yml:1]:
```suggestion
```

File: alert-policies/ext-hcp-vault/sealed-status.yml [hunk 9]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (aswanson-nr) [inline alert-policies/ext-hcp-vault/sealed-status.yml:1]:
```suggestion
```

**Review by aswanson-nr — COMMENTED** [review]
Hey @IrAriasKC, this looks good to me! 
We can get rid of those document separators in the alerts, I'll add that commit and get this merged!

File: alert-policies/ext-hcp-vault/handle-login-request.yml (hunk 7 above)
Comment (aswanson-nr) [inline alert-policies/ext-hcp-vault/handle-login-request.yml:1]:
```suggestion
```

File: quickstarts/ext-hcp-vault/config.yml [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+slug: ext-hcp-vault
+
+# Displayed in the UI (required)
+title: HCP Vault
+
+# Long-form description of the quickstart (required)
+description: |
+  The HCP Vault quickstart allows you to get visibility into the performance and available of your HCP Vault service and dependencies. Use this quickstart together with the mock up integrations.
+
+ ## HCP Vault quickstart highlights
```
Comment (aswanson-nr) [inline quickstarts/ext-hcp-vault/config.yml:11]:
```suggestion
  ## HCP Vault quickstart highlights
```

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**